test/migrators/division/always_division.hrx

```diff
@@ -0,0 +1,37 @@
+<==> input/entrypoint.scss
+@function six-divided-by-three() {
+  @return 6px / 3px;
+}
+
+a {
+  b: (4px + 2px) / 3px;
+  c: 6px/3px + 1;
+  d: (6px / 3px);
+  e: 6px / (2px + 1px) / 2;
+  $f: 6px / 3px;
+
+  // * and % always return numbers, so we can assume division here
+  g: (3 * $x) / 4;
+  h: (3 / $x) % 4;
```

Comment:
```suggestion
  h: 3 / $x % 4;
```

Right now, this `/` will be transformed into `divide()` regardless of the operator after it, because of the parentheses.